### ISSUE TITLE
[BUG] [3.1.2] Align Columns

### ISSUE THREAD

**Issue by chsk-uk** [texstudio-org/texstudio#1718]
Good morning,

the _Align Columns_ functionality for tabular environments in TeXstudio 3.1.2 (current version) will mistakenly comment out commands such as `\midrule` when these are preceded by a comment. Please see the MWE below.

### How to reproduce

MWE:

``` 
\documentclass{article}
\usepackage{booktabs}
\begin{document}
\begin{tabular}{l}
Test\\
% Comment
\midrule
Test
\end{tabular}
\end{document}
```

To reproduce:

1. Copy the above code into a new file (editor window), and 
2. Click on _Align Columns_ (Ctrl-Alt-C).

### Expected behavior

Expected output:

```
\documentclass{article}
\usepackage{booktabs}
\begin{document}
\begin{tabular}{l}
	Test \\
% Comment
\midrule
	Test
\end{tabular}
\end{document}
```

(or some variation thereof)

### Actual behavior

Actual output:

```
\documentclass{article}
\usepackage{booktabs}
\begin{document}
\begin{tabular}{l}
	Test \\
% Comment\midrule
	Test
\end{tabular}
\end{document}
```

Thanks for looking into/fixing this, and as always, thank you very much for developing TeXstudio!

**Comment by sunderme** [comment]
fixed with 6825c69d3345ece832191e7e00c25a7fd72c2619

**Comment by chsk-uk** [comment]
Excellent. Thank you very much for the quick fix!
